core/src/main/java/com/klaytn/caver/methods/response/KlayRewards.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2022 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.methods.response;
+
+import java.util.Map;
+
+import org.web3j.protocol.core.Response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class KlayRewards extends Response<KlayRewards.BlockRewards> {
+    public static class BlockRewards {
+        /**
+         * the amount newly minted
+         */
+        @JsonProperty("minted")
+        private String Minted; 
```

Comment:
The member field in java use lower camel case in general. It is not like golang which use the first capital letter in open field. So that Minted change to minted will better.

Reply:
Resolved it, thanks